target/linux/ramips/base-files/etc/hotplug.d/firmware/10-rt2x00-eeprom

```diff
@@ -41,6 +41,12 @@ board=$(board_name)
 case "$FIRMWARE" in
 "soc_wmac.eeprom")
 	case $board in
+	dwr-921-c1)
```

Comment:
Should be like bellow?
`dlink,dwr-116-a1|\`
`dlink,dwr-921-c1)`

Reply:
I set the board name dwr-921-c1 in the target/linux/ramips/base-files/lib/ramips.sh

Reply:
I set the "dwr-921-c1" name in target/linux/ramips/base-files/lib/ramips.sh